google/cloud/bigtable/tools/convert_acceptance_tests.py

```diff
@@ -61,18 +61,28 @@ def camel_case(s):
 
 
 def print_test(t):
-    o = '// Test name: "' + t["name"] + '"\n'
-    o += "TEST_F(AcceptanceTest, " + camel_case(t["name"]) + ") {\n"
+    o = '// Test name: "' + t["description"] + '"\n'
+    o += "TEST_F(AcceptanceTest, " + camel_case(t["description"]) + ") {\n"
 
     o += "  std::vector<std::string> chunk_strings = {\n"
-    for c in t["chunks"]:
-        s = c.rstrip()
-        s = s.replace("<\n ", "<")
-        s = s.replace("\n>", ">")
-        s = s.replace("\n", "\n          ")
-        o += '      R"chunk(\n'
-        o += "          " + s + "\n"
-        o += '        )chunk",\n'
+    if ('chunks' in t):
```

Comment:
i think we previously parsing json element and adding to string, whereas new logic we are restricting to certain keys. 

it is better to use a generic approach rather then key based.

Reply:
> i think we previously parsing json element and adding to string, whereas new logic we are restricting to certain keys.
> 
> it is better to use a generic approach rather then key based.

Hi, 'chunks' in previous json file are implemented using python 'list of strings' whereas in the new json file 'chunks' are implemented as python 'dictionary' objected and hence parsed based on key-value approach. Hope it justifies the new implementation.

Reply:
what about converting dictionary to a list and then parsing it. That way we can accommodate any new addition in chunks?